report/Titelblad.tex

```diff
@@ -33,7 +33,16 @@
   \includegraphics[width=0.9\textwidth]{images/aau_logo.pdf}\\[1.0 cm]
  \end{flushright}
  \begin{flushleft}
-  \textbf{Abstract:}\\INSERT HERE
+  \textbf{Abstract:}\\
+  In the danish education system, the Arduino platform is often used to introduce the students to computer science. 
```

Comment:
danish -> Danish
also i prefer educational system, compared to education system.